### ISSUE TITLE
Missing SecAccessControlCreateWithFlags for iOS KeyChain query

### ISSUE THREAD

**Issue by netsrac101** [juliansteenbakker/flutter_secure_storage#836]
It is not possible to set additional security on KeyChain items to ensure users presence.
See: https://developer.apple.com/documentation/security/secaccesscontrolcreatewithflags(_:_:_:_:)

e.g.
 let access = SecAccessControlCreateWithFlags(
            nil,
            kSecAttrAccessibleWhenUnlockedThisDeviceOnly,
            .userPresence,
            nil
        ) 

You hand those flags over in kSecAttrAccessControl attribute on a keychain query.

Why can this be a security risk not to hand those flags over?
https://stackoverflow.com/questions/69752685/understanding-the-access-control-setting-userpresence-for-keychain-items
TL;DR: There is especially a security risk for jailbroken devices.
